### DISCUSSION TITLE
Geogig vs. Kart

### DISCUSSION THREAD

**Discussion by ffischer1984** [koordinates/kart#865] [Q&A] [answered]
Hi,

has somebody evaluated the advantages and disadvantages between geogig and kart?
What works better in kart and what in geogig.

- geogig
  - advantages
    - has a geonode/geoserver plugin
  - disadvantages
     -  looks a little bit abandoned to me (sorry)
- kart
   - advantages
      - will have tile/rasterdata support 
      - seems under active developerment
  - disadvantages:
      - has more open issues??

**Comment by rcoup** [comment, marked as answer]
Geogig hasn't been actively developed for several years now: Boundless stopped developing it sometime after the Planet acquisition. Geogig is part of the "inspired by Git" camp of solutions for data versioning — we've taken the approach with Kart to build _on top of_ Git: meaning we can focus on the data side, rather than spending our time & effort reinventing the building blocks that Git already has in place. Our [presentation at PostGIS Day 2022](https://youtu.be/_7ETwmMlUtY) is available as a video (unfortunately the last couple of FOSS4G videos aren't available), and there's some discussion of how we're approaching it at [5mins](https://youtu.be/_7ETwmMlUtY?t=304)

> has a geonode/geoserver plugin

We'll get there eventually :-) 

> disadvantages:
> - has more open issues??

We're open about what Kart does and doesn't do right now, and we're continuously developing and improving Kart over time.

I'll transfer this to a discussion for any further followups
